### DISCUSSION TITLE
How can i get the list of all currently executing Jobs in Cluster

### DISCUSSION THREAD

**Discussion by fsaffioti** [quartznet/quartznet#1352] [Q&A] [answered]
GetCurrentlyExecutingJobs() seems to be working only in a single node
How can I get all currently executing jobs from all nodes?
Searching for it someone says that i should querying Quartz table qrtz_fired_triggers directly.

Is that true?

**Comment by lahma** [comment, marked as answer]
Like you've discovered, there's currently no API for that. If you go with direct DB query route, make sure you don't take any locks that might infer with Quartz logic.
